sushi-config.yaml

```diff
@@ -58,11 +58,15 @@ pages:
       title: AU eRequesting Data for Interoperability Mappings
   use-cases.md:
     title: Use Cases
-  profiles-and-extensions.md:
-  capability-statements.md: 
-    title: Actors and Capabilities
-  search-parameters-and-operations.md:
-  terminology.md:
+  fhirartifacts.md:
+    title: FHIR Artefacts
+    artifacts.html:
+      title: Artefacts Summary
+    profiles-and-extensions.md:
+    capability-statements.md: 
+      title: Actors and Capabilities
+    search-parameters-and-operations.md:
+    terminology.md:
   examples.md:
   support.md:
```

Comment:
Commenting here to get an inline comment: 
In the Table of Contents: 
1. The entries under 3 Guidance should be reordered to match the order they appear in the main menu, e.g. 3.4 AU eRequesting Data for Interoperability Mappings becomes 3.1 eRequesting Data for Interoperability Mappings
2. The entries under 5 FHIR Artefacts should be reordered to match the order they appear in the main menu, e.g. Terminology comes before Actors and Capabilities. 
Please remove `Search Parameters and Operations`, there is no such page in this IG.
3. Licence under 7 Support should be 'License and Legal'.